hermes-console-vue/src/views/admin/stats/StatsView.vue

```diff
@@ -0,0 +1,111 @@
+<script async setup lang="ts">
+  import { useI18n } from 'vue-i18n';
+  import { useStats } from '@/composables/use-stats/useStats';
+
+  const { t } = useI18n();
+
+  const {
+    topicCount,
+    ackAllTopicCount,
+    ackAllTopicShare,
+    trackingEnabledTopicCount,
+    trackingEnabledTopicShare,
+    avroTopicCount,
+    avroTopicShare,
+    subscriptionCount,
+    trackingEnabledSubscriptionCount,
+    trackingEnabledSubscriptionShare,
+    avroSubscriptionCount,
+    avroSubscriptionShare,
+    loading,
+    error,
+  } = useStats();
+</script>
+
+<template>
+  <v-container>
+    <v-row class="home__logo" justify="center" cols="12">
+      <v-col md="12">
+        <loading-spinner v-if="loading" />
+        <console-alert
+          v-if="error"
```

Comment:
Hmm, i'm not sure how this works, but error is a structure of errors so checking if error is not null could lead to returning always true (?) as error structure is not null. I've checked, and it seems like it's always true when all the errors are nulls. Code below logs 'Error occurrence'
```typescript
interface SomeErrors {
    fetchError: Error | null;
    updateError: Error | null;
}

const errors = {
    fetchError: null,
    updateError: null,
} as SomeErrors;

if (errors) {
    console.log('Error occurrence');
} else {
    console.log('No error detected');
}
```

Reply:
changed to single error